python/plugins/processing/algs/lidar/fusion/FilterData.py

```diff
@@ -71,8 +71,5 @@ def processAlgorithm(self, progress):
             FusionUtils.createFileList(files)
             commands.append(FusionUtils.tempFileListFilepath())
         FusionUtils.runFusion(commands, progress)
-        commands = [os.path.join(FusionUtils.FusionPath(), 'LDA2LAS.exe')]
-        commands.append(outFile)
-        commands.append(self.getOutputValue(self.OUTPUT))
```

Comment:
if there is no need to convert to LAS anymore, the two lines below should be removed as well